frontend/webapp.js

```diff
@@ -67,7 +70,7 @@ app.get("/admin/subscriptions/:id", (req, res) => {
 });
 
 app.get("/email", (req, res) => {
-  res.send({ email: req.user });
+  res.send({ email: "darwin@yelp.com" });
```

Comment:
Is this a change that was supposed to be for testing only? Hardcoing darwin here doesn't seem to make sense.

Reply:
yeah, I forgot to remove that. It was just for testing.